.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: CI Tests
+
+on: [push]
+
+jobs:
+  release:
+   runs-on: ubuntu-latest
+   env:
+     CI: true
```

Comment:
This is a placeholder. I'm not sure if we need to specify an environment for the test build.